test/support/api/patients.js

```diff
@@ -92,3 +92,18 @@ Cypress.Commands.add('routePatientSearch', (mutator = _.identity) => {
   })
     .as('routePatientSearch');
 });
+
+Cypress.Commands.add('routePatientEngagementStatus', (mutator = _.identity) => {
```

Comment:
I think this can be `(status = _.sample(this.fxPatientEngagementStatuses)) =>`  then I think `cy.routePatientEngagementStatus(status)` can be added to `routePatient` with `routePatient accepting `mutator, status`

Reply:
Unfortunately the engagement status command needs to establish the fixture data on its own because there are a handful of scenarios where `routePatient` isn't called (flows and actions).

Reply:
To this point though.. In _every_ instance where patient engagement status would be needed, patient _must_ be loaded somewhere..  _so_ it seems like either this could be added to `routePatient` and whatever the other route is, and then recalled very infrequently.. or we could just not add it _everywhere_ and let it fail unless we're testing it.   We're adding `routePatientEngagementStatus` 15 times, but probably only need it.. once or twice.